apps/mosaico/src/Mosaico.purs

```diff
@@ -230,6 +236,27 @@ mosaicoComponent initialValues props = React.do
 
   pure $ render setState state initialValues.components initialValues.nav onPaywallEvent
 
+sendArticleAnalytics:: Article -> Maybe User -> Effect Unit
```

Comment:
I'd move this function to the `Analytics` module, as it kinda only contains analytics logic (= convert types into analytics types and push to data layer)